tests/commerce_integration/TEC/Admin/__snapshots__/Ticket_NoticesTest__messages_should_match with data set 0__1.php

```diff
@@ -0,0 +1 @@
+<?php return '<p>Order #123 has been marked as refunded. Remember to process the refund in your payment gateway.</p>';
```

Comment:
All snapshots would need to be deleted after switching to Luca's and reproduced.